pkgs/development/python-modules/hug/default.nix

```diff
@@ -20,19 +20,29 @@ buildPythonPackage rec {
     sha256 = "05rsv16g7ph100p8kl4l2jba0y4wcpp3xblc02mfp67zp1279vaq";
   };
 
-  nativeBuildInputs = [ pytestrunner ];
   propagatedBuildInputs = [ falcon requests ];
 
-  checkInputs = [ mock marshmallow pytest numpy ];
-  checkPhase = ''
-    mv hug hug.hidden
-    # some tests attempt network access
-    PATH=$out/bin:$PATH pytest -k "not (test_request or test_datagram_request)"
+  checkInputs = [ mock marshmallow pytestCheckHook pytestrunner numpy ];
```

Comment:
Isn't pytestrunner deprecated?

Reply:
I tried to remove it and the build stopped working with this error during `setuptoolsBuildPhase`:
```
ERROR: Could not find a version that satisfies the requirement pytest-runner
ERROR: No matching distribution found for pytest-runner
```

Reply:
I think in other packages we usually remove that dependency.

Reply:
With patching? Could you point me to an example?

Reply:
@SuperSandro2000:  I added a `postPatch` hook that removes the pytest-runner dependency. PTAL